src/schema.graphql

```diff
@@ -36,6 +37,8 @@ type Mutation {
     deleteEvent(id: ID!): Event
     addEventSubscriber(id: ID!): Event
     removeEventSubscriber(id: ID!): Event
+    createEventComment(id: ID!, content: String!): Comment
+    deleteEventComment(id: ID!): Comment
```

Comment:
see `datamodel.prisma` for better implementation, perhaps with `createComment(eventId: ID, content: String!)` so that it is expandable to `createComment(eventId: ID, [someOtherId]: ID, ...`. Also make sure to programatically throw errors on `eventId` nullity